src/controllers/webviewController.ts

```diff
@@ -60,7 +60,7 @@ export class WebviewPanelController implements vscode.Disposable {
                     // give a 2 sec delay because the the webview visible event is fired
                     // before the angular component is actually built. Give time for the
                     // angular component to show up
-                    setTimeout(async () => await this.queryRunner.refreshQueryTab(), 1700);
+                    setTimeout(async () => await this.queryRunner.refreshQueryTab(), 2000);
                 }
```

Comment:
Why? How do you know this new value is high enough?

Reply:
This came from testing myself initially from the initial PR, and hence the comment on the top. I made some perf changes, which led it query results loading faster, but it takes more time on script as select, for example